hicommonwealth/commonwealth
# Eliminate CommunityBanners table.

## Link to Issue
Closes: #8789 

## Description of Changes
- Adds `banner_text` to Communities to permit removal of CommunityBanners table.
- Wrote migration to move existing banner data.
- Route outputs / UI not modified.

## Test Plan
- Run migration, ensure no data loss (checking CommunityBanners table pre-migration against banner_text on Communities post-migration). Undo migration to ensure no data loss (reversibility). Re-run migration.
- Confirm banners still load. Confirm a banner can be added to a community. Confirm a banner can be removed from a community. Confirm a banner can be edited.
